### PR TITLE
Languagetool second try

### PR DESCRIPTION
**Pull request by 1Euro7Cent**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your server use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

File: software/languagetool/egg-languagetool.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            "container": "ghcr.io\/pterodactyl\/installers:alpine",
+            "entrypoint": "ash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Include traindata",
+            "description": "This takes up a lot of disk space. I'm talking about multiple gigabytes. \r\n\r\nSeparate each value with spaces. For example, \"de en\". \r\n\r\nView all available models at https:\/\/languagetool.org\/download\/ngram-data\/",
```
Comment (parkervcp) [inline software/languagetool/egg-languagetool.json:33]: Could probably just go with a 

"This uses many gigabytes of space"

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: software/languagetool/egg-languagetool.json (hunk 1 above)
Comment (1Euro7Cent) [reply software/languagetool/egg-languagetool.json:33]: instead of `This takes up a lot of disk space. I'm talking about multiple gigabytes.`?

**Comment by 1Euro7Cent** [comment]
How is it going? It seems like you forgot that

**Comment by 1Euro7Cent** [comment]
@parkervcp hello?

**Comment by parkervcp** [comment]
> @parkervcp hello?

I am sorry I haven't had the time to look at this, Honestly I get to this when I can.

**Review by parkervcp — APPROVED** [review]

**Pull request merged**